one-sdk/src/main/java/tv/accedo/one/sdk/implementation/IfModifiedTask.java

```diff
@@ -26,7 +27,7 @@
     private static final String FILENAME_VERSIONCODE = "OneSdkVersion";
     private static final int LAST_CACHEBREAKING_VERSION_UPDATE = 100; //1.0(.0)
 
-    private static final SimpleDateFormat sdfIfModifiedSince = new SimpleDateFormat("EEE, dd MMM yyyy HH:mm:ss zzz", Locale.US);
+    private static final SimpleDateFormat sdfIfModifiedSince = new SimpleDateFormat("EEE, dd MMM yyyy HH:mm:ss zzz", Locale.getDefault());
```

Comment:
Are you sure default is the good value here? Could you please test setting your device to different locales? (like changing device language to german, polish, whatnot)

Reply:
I think, Locale.US was set on purpose, Date format in HTTP header is very strict because of the backward compatibility.  Few related materials:
- https://stackoverflow.com/a/21121047
- https://tools.ietf.org/html/rfc2616#section-3.3
- https://tools.ietf.org/html/rfc7231#section-7.1.1.1

Reply:
tried with different and found the format date will be different language if the locale is set as Local.getDefault() 
like for Germany, it will be Mi., 18 März 2020 07:49:26 GMT+00:00
so set it to Local.US.
and tried again with setting different device locale and its working fine.

Reply:
That looks good to me! If @siczmj also approves, i'm fine with releasing this as 1.0.2!